azkaban_cli/azkaban.py

```diff
@@ -409,3 +433,124 @@ def fetch_projects(self):
         self.__catch_login(response)
 
         return response.text
+
+
+    def add_permission(self, project, group, permission_options):
+        """Add permission command, intended to make the request to Azkaban and treat the response properly.
+
+        This method receives the project name, the group name, and the permission options and execute 
+        request to add a group permission to the project and evaluate the response.
+
+        :param project: Project name on Azkaban
+        :type project: str
+
+        :param group: Group name on Azkaban
+        :type project: str
+
+        :param permission_options: The group permissions in the project on Azkaban
+        :type project: Dictionary
+        """
+
+        self.__check_if_logged()
+
+        permission_options = self.__check_group_permissions(permission_options)
+
+        response = api.add_permission_request(
+            self.__session,
+            self.__host,
+            self.__session_id,
+            project,
+            group, 
+            permission_options
+        )
+
+        self.__catch_response_error_ignore_empty(response, AddPermissionError)
+        
+        logging.info('Group [%s] add with permission in the Project [%s] successfully' % (group, project))
+
+    def remove_permission(self, project, group):
+        """Remove permission command, intended to make the request to Azkaban and treat the response properly.
+
+        This method receives the project name and the group name and execute 
+        request to remove a group permission from the project and evaluate the response.
+
+        :param project: Project name on Azkaban
+        :type project: str
+
+        :param group: Group name on Azkaban
+        :type project: str
+
+        """
+
+        self.__check_if_logged()
```

Comment:
I think no, 

Because `@login_required` is a function in the CLI layer and have specific features to interact using the CLI if the user isn't logged in.